vllm/README.md

```diff
@@ -22,3 +22,28 @@ Once the server is deployed, you can use the `yichat.py` script to chat with the
 ```bash
 python yichat.py
 ```
+
+## Multi-modal inference
+
+We can deploy a multi-modal model like Phi-3.5-vision-instruct. The `phi.py` script demonstrates how to do this.
+
+```bash
+beam deploy phi.py:phi
+```
+
+Once the server is deployed, you can use the `phichat.py` script to chat with the model.
```

Comment:
can we avoid using the model name in the file name? I feel like it's more confusing than having something like e.g. inference-server.py and chat.py